content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -525,11 +525,13 @@ This means that support is still available but will be removed in a future Kuber
 Older versions of Kubernetes also supported the following in-tree PersistentVolume types:
 
 * [`awsElasticBlockStore`](/docs/concepts/storage/volumes/#awselasticblockstore) - AWS Elastic Block Store (EBS)
-  (**not available** in v1.27)
+  (**not available** starting v1.27)
```

Comment:
Hello @adityasamant25 .

What do you think about adding a preposition or some other content after the `not available` phrase,
`not available starting with v1.27` or `not available after v1.26` or `not available in v1.27 or later`

Reply:
Sure thing.